scripts/exrrfs_ioda_bufr.sh

```diff
@@ -70,6 +75,18 @@ for yaml in "${yaml_list[@]}"; do
 done
 
 # --------------------------------------------------
+# run  bufr2ioda tool for atms bufr obs
+# --------------------------------------------------
+${cpreq} "${FIXrrfs}/atms_beamwidth.txt" .
+${cpreq} "${PARMrrfs}/bufr_atms_mapping.yaml" .
+input_file="atmsbufr"
+output_file="ioda.atms_{splits/satId}.nc"
```

Comment:
@xyzemc just a quick check, for line 83, Is this should be "ioda.atms_{splits/satId}.nc" or "ioda_atms_{splits/satId}.nc"? 
Thanks!